test/unit/ajax.js

```diff
@@ -2034,6 +2034,25 @@ module( "ajax", {
 		});
 	});
 
+	asyncTest( "jQuery[get|post]( config ) - simple with xml", 2, function() {
+		jQuery.when.apply(jQuery,
```

Comment:
The same remark about spaces; there should be a space after `(`, `[` and before `)`, `]` etc.